From b23652d3238389aae3c6ddbaab85bc7468a3fb6a Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Thu, 28 Aug 2014 14:41:48 +0200
Subject: [PATCH] add support for ppc64le

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 79d69e2..eea430d 100755
--- a/configure
+++ b/configure
@@ -17425,7 +17425,7 @@ $as_echo_n "checking for boostlib >= $boost_lib_version_req... " >&6; }
                         libsubdirs="lib"
     ax_arch=`uname -m`
     case $ax_arch in
-      x86_64|ppc64|s390x|sparc64|aarch64)
+      x86_64|ppc64|ppc64le|s390x|sparc64|aarch64)
         libsubdirs="lib64 lib lib64"
         ;;
     esac
-- 
1.9.3

